COLLECTION-ROUTING.md

```
@@ -0,0 +1,270 @@
+# Force github content into cortex
+axon github jmagar/axon_rust --collection cortex
+
+# Force a session into a custom collection
+axon sessions --collection my-custom-collection
+```
+
+---
+
+## Derivation Logic
+
+### `github` — trivial
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)